### ISSUE TITLE
Truncation does not always produce a small-enough payload

### ISSUE THREAD

**Issue by brianr** [rollbar/rollbar-gem#864]
Example seen from rollbar-gem 2.12.0:

```
Failsafe from rollbar-gem. Could not send payload due to it being too large after truncating attempts. Original size: 81038972 Final size: 1386110
```

We ought to at least be able to report *something* more useful than that. Let's try to figure out what kind of situation this might have been, and figure out a truncation strategy that would produce something useful.

**Comment by letaniaferreira** [comment]
another one - https://app.intercom.io/a/apps/rlisg82q/inbox/inbox/488687/conversations/22216872441

**Comment by waltjones** [comment]
https://github.com/rollbar/rollbar-gem/pull/872 released in 2.20.1 does more to address this, by also removing the contents of `request` and then `extra` if needed.
